models/articles.models.js

```diff
@@ -37,15 +39,25 @@ exports.selectArticles = (sort_by = "created_at", order = "desc") => {
       msg: "Bad request",
     });
   }
-  return db
-    .query(
-      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, 
-      articles.article_img_url, CAST(COUNT(comments.article_id) as INT) AS comment_count FROM articles LEFT OUTER JOIN comments ON 
-      articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.${sort_by} ${order}`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+
+  let sqlQuery = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, 
```

Comment:
Maybe spend some time formatting this SQL string. It's not imperative, but it can certainly make your project more presentable :)